core/esmf-aspect-model-document-generators/src/main/java/org/eclipse/esmf/aspectmodel/generator/diagram/AspectModelDiagramGenerator.java

```diff
@@ -180,7 +186,17 @@ private void generatePng( final String dotInput, final OutputStream output ) thr
       final Graphviz graphviz = Graphviz.fromGraph( g );
       graphviz.render( guru.nidi.graphviz.engine.Format.PNG ).toOutputStream( output );
    }
-
+   private File generateTmpFontFile() throws IOException {
+      File tempFontFile = new File( System.getProperty( "java.io.tmpdir" ) + "/aspect-model-diagram.tmp" );
```

Comment:
Please use `File.separator` instead of the hardcoded `/` (to have it work on Windows)